qiskit_experiments/framework/composite/batch_experiment.py

```diff
@@ -79,7 +94,10 @@ def circuits(self):
                 # Remap qubits if required
                 if qubit_mapping:
                     circuit = self._remap_qubits(circuit, qubit_mapping)
```

Comment:
I think I disagree here, but it will be good to also hear @chris' opinion.
Suppose that I have a batch experiment with two experiments, that are not independent. The first experiment runs something on qubits 0 and 1, and the second experiment runs something on qubit 1 only, and from both the parent batch experiment deduces something say about the cross-talk between qubits 0 and 1.
When the user draws the circuits, I want the user to see that the second experiment applies on the second qubit of the first experiment.
We can think of writing a draw method, that draws the circuits without remapping them and also prints the mappings. In my example, it will draw the circuits of the second experiment with qubit 0, but each of these drawings will be accompanied with text, something like `0 -> 1`.

Reply:
If, eventually, we decide not to remap in `circuits`, then, as usual, this is to be done in another PR.

Reply:
umm... this is good point, however, if two experiments are independent and with different qubit set, this returns meaningless (or very confusing) circuit, because qubit index is just determined by the ordering of sub experiment. The remapping is indeed an overhead because it regenerates new circuits with larger qubit set. Updating the drawer label wihtout applying layout makes sense.

Reply:
@nkanazawa1989 We define an experiments physical qubits to be the number of qubits for all circuits generated. If you have a batch experiment where each component experiment has a different number of qubits, then each batch experiment circuit generated should be defined on those qubits, which is why remapping may need to be done. This remapping is also necessary for batch experiments to compose correctly with other batch or parallel experiments when these qubits are not all the same in each component experiment

@nkanazawa1989